### PR TITLE
Detailed view for task

### PR DESCRIPTION
**Pull request by pbochis**
- removed description from task card
- created detailed view for a task

### CODE REVIEW

**Comment by pbochis** [comment]
Fixes #61 except for the hero animation, which is p2 currently

File: app/elements/task/task-card.html [hunk 1]
```diff
@@ -43,17 +53,19 @@
 			class="card"
 			on-mouseover="onHovered"
 			on-mouseout="onUnhovered"
-		>
+		  >
```
Comment (victorbalan) [inline app/elements/task/task-card.html]: random spaces

File: app/elements/task/task-card.html [hunk 2]
```diff
@@ -43,17 +53,19 @@
...
 			<h2>{{task.Name}}</h2>
 			<template is="dom-if" if="{{showButton}}">
-				<paper-fab mini icon="add" on-click="toggleSelectTask"></paper-fab>
+				<paper-fab class="task-selector" icon="add" on-click="toggleSelectTask"></paper-fab>
```
Comment (victorbalan) [inline app/elements/task/task-card.html]: `toggleSelectTask` should be private

File: app/elements/routing.html [hunk 3]
```diff
@@ -83,6 +83,16 @@
 		});
 	});
 
+	page('/tasks/:taskKey', function(data) {
+		checkAuthorization();
+		checkIsCompany();
+		app.header = 'Tasks';
```
Comment (victorbalan) [inline app/elements/routing.html]: Wrong header

**Pull request merged**